src/ostorlab/agent/mixins/agent_open_telemetry_mixin.py

```diff
@@ -42,13 +45,18 @@ def get_trace_exporter(self) -> sdk_export.SpanExporter:
         The urls are customized to respect the following format:
             name_of_the_tracing_tool:hostname:port
             eg: jaeger:jaeger-host:8631
+            for gcp the format is:
+            name_of_the_tracing_tool:project_id:service_account_json_path
+            eg: gcp:project_1:/tmp/key.json
```

Comment:
we should have the json content in the URL, we should not rely on external paths for passing credentials.